lib/runners/processor/fxcop.rb

```diff
@@ -0,0 +1,98 @@
+module Runners
+  # A processor class for FxCop Analyzers
+  # https://docs.microsoft.com/en-us/visualstudio/code-quality/install-fxcop-analyzers
+  # This is a beta release. We support only repositories that has .csproj file directly under analysis root directory.
+  class Processor::FxCop < Processor
+    #parameters
+    ANALYSIS_LOGFILE_PATH = '/tmp/sider_analysis_out.json'.freeze
+    RULE_ID_PATTERN = /CA[0-9]+/.freeze
+
+    Schema = StrongJSON.new do
+      let :issue, object(
+        severity: string?,
+        )
+    end
+
+    def self.ci_config_section_name
+      'fxcop'
+    end
+
+    def analyzer_name
+      'FxCop'
+    end
+
+    def analyzer_version
+      ENV['FXCOP_VERSION']
+    end
+
+    def setup
+      # restore project (install dependencies)
+      _, _, status = capture3('dotnet', 'restore')
+      unless status.success?
+        msg = <<~EOS
+          Failed to restore .NET Core Project. You have to put project file (.csproj) under the analysis root. Please check the project structure and 'root_dir' parameter in #{ci_config_path_name}.
+          See: https://help.sider.review/getting-started/custom-configuration#root_dir-option
+          Note: We only support a project managed with MSBuild. No support for other build systems. (e.g. Nuke, Cake)
+        EOS
+        return Results::Failure.new(guid: guid, message: msg, analyzer: analyzer)
+      end
+
+      # install FxCop Analyzers from nuget Repository
+      cmdline_add_analyzer = ['dotnet', 'add', 'package', 'Microsoft.CodeAnalysis.FxCopAnalyzers', '--version', analyzer_version]
```

Comment:
To directly pass arguments to `capture3!` looks shorter.

```suggestion
      capture3!('dotnet', 'add', 'package', 'Microsoft.CodeAnalysis.FxCopAnalyzers', '--version', analyzer_version)
```